rocketpy/utilities.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Franz Masatoshi Yuri"
+__copyright__ = "Copyright 20XX, Projeto Jupiter"
+__license__ = "MIT"
+
+import re
+import math
+import bisect
+import warnings
+import time
+from datetime import datetime, timedelta
+from inspect import signature, getsourcelines
+from collections import namedtuple
+
+import numpy as np
+from scipy import integrate
+from scipy import linalg
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+
+from rocketpy import Environment, Rocket, SolidMotor, Flight
+
+
+def compute_cds_from_drop_test(final_speed, final_air_density, rocket_mass, gravity):

```

Comment:
I may be wrong, but is it more common to find `CdS` in the literature, instead of `cds`?
Would `terminal_velocity` be a more technical term when compared with `final_speed`?
Would it be a bad idea to add default values for the air density and gravity?

```suggestion
def compute_CdS_from_drop_test(terminal_velocity, rocket_mass, air_density=1.225, gravity=9.80665):

```

Reply:
Indeed CdS is the most precise way to write, I just wrote "cds" because I thought it would be best to use only letters in lowercase.